test/service/storage/elasticsearch.test.js

```diff
@@ -3536,7 +3536,7 @@ describe('Test: ElasticSearch service', () => {
       elasticsearch._mExecute = sinon.stub().resolves(mExecuteResult);
     });
 
-    it('should call _mExecute with formated documents', () => {
+    it('should call _mExecute with formated documents', async () => {
```

Comment:
Useless async keyword
```suggestion
    it('should call _mExecute with formated documents', () => {
```